src/database/migrations/20200506192302-create-user-table.js

```diff
@@ -7,15 +7,15 @@ module.exports = {
       type: Sequelize.UUID,
       defaultValue: Sequelize.UUIDV4
     },
-    firstName: {
+    surName: {
       allowNull: false,
       type: Sequelize.STRING
     },
-    surname: {
+    firstName: {
       allowNull: false,
       type: Sequelize.STRING
     },
-    middleName: {
+    lastName: {
       allowNull: true,
       type: Sequelize.STRING
     },
```

Comment:
also, surname and last name mean the same thing so let's eliminate one

Reply:
Okay, noted